src/views/language/LanguageTable.tsx

```diff
@@ -112,6 +159,112 @@ const LanguageTable: React.FC = () => {
           sortParam: SortBy.PopulationOfDescendents,
           columnGroup: 'Population',
         },
+        {
+          key: 'Vitality (Meta)',
+          label: 'Vitality (Meta)',
```

Comment:
1) the label is optional if its identical to the key
2) For the Metascore I'd just call this 'Vitality'.

```suggestion
          key: 'Vitality',
```